app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -14,6 +15,12 @@ module HyraxHelperBehavior
     include Hyrax::ChartsHelper
     include Hyrax::DashboardHelperBehavior
 
+    # Which translations are available for the user to select
+    # @return [Hash<String,String>] locale abbreviations as keys and flags as values
+    def available_translations
+      { 'en' => '🇬🇧', 'es' => '🇪🇸' }
```

Comment:
I'd consider this approach when there are only two languages:

![bassi-veratti_collection](https://cloud.githubusercontent.com/assets/101482/21195453/141dd2a6-c1f1-11e6-9d4e-0304604fa78a.png)

And when there are three or more, make it a dropdown, showing the current language as the label and the other available languages as the menu options.